File: Printpress.UI/Printpress_Angular/src/app/features/order/models/orderSellingItem/order-selling-item-get.dto.ts
import { IObjectState } from '../../../../core/models/i-object-state';

export interface OrderSellingItemGetDto extends IObjectState {
    id: string;
    name: string;
    orderId: string;
    inventoryItemId?: string;
    isInventoryItem: boolean;
    quantity: number;
    price: number;
    inventoryItemName?: string;
}

File: Printpress.Application/Orders/DTO/Order/Get/OrderSellingItemGetDTO.cs
using Printpress.Domain;

namespace Printpress.Application;

public class OrderSellingItemGetDTO : TrackedDTO
{
    public Guid Id { get; set; }
    public string Name { get; set; }
    public Guid OrderId { get; set; }
    public Guid? InventoryItemId { get; set; }
    public bool IsInventoryItem { get; set; }
    public int Quantity { get; set; }
    public decimal Price { get; set; }
    public string? InventoryItemName { get; set; }
}
